src/mscorlib/src/System/Buffer.cs

```diff
@@ -262,7 +262,7 @@ internal unsafe static void Memmove(byte* dest, byte* src, nuint len)
         {
 #if AMD64 || (BIT32 && !ARM)
             const nuint CopyThreshold = 2048;
-#else
+#elif !ARM64
```

Comment:
glibc optimizations have not been incorporated in Linux OS distros.  Expected possibly with Ubuntu 18.04.

Windows Arm64 may need a slightly different optimization.  
`#elif (WINDOWS || !ARM64)` ?

Reply:
Maybe we should have `USE_OS_MEMMOVE` define

Reply:
Also, it would be nice to have a comment in the code on why we are not using the OS impl on Unix.